javascript-practice/src/styles/main.css

```diff
@@ -3,6 +3,7 @@
 @import "./components/index.css";
 @import "./layouts/index.css";
 @import "./pages/login.css";
+@import "./modal/index.css";
```

Comment:
Why do you have folder for modal? Should we use it as components?

Reply:
Btw, I found you have NO CODE for modal. Should not add empty file until you use it.